Documentation/configuration/metrics.rst

```diff
@@ -41,7 +41,18 @@ Endpoint
 Datapath
 --------
 
-* ``datapath_errors_total``: Total number of errors occurred in datapath management, labeled by area, name and address family.
+* ``datapath_errors_total``: Total number of errors occurred in datapath
+  management, labeled by area, name and address family.
+* ``datapath_conntrack_gc_runs_total``: Number of times that the conntrack
+  garbage collector process was run. It contains a label status that describes
+  if it was successful or not.
+* ``datapath_conntrack_gc_key_fallbacks_total``: Number of times that the Key fallback
```

Comment:
I'm wondering since this is a "per-entry" type statistic (may occur multiple times per GC run), perhaps it also makes sense to track in the same metric as "datapath_conntrack_gc_size" ?

Reply:
@raybejjani ^^^^

Reply:
Discussed this briefly yesterday and the worry was mostly that the scale of key_fallbacks_total is very different from the total size. The total size may be 200K while the keyfallbacks maybe 0..10

Reply:
We could clearly stuff everything into a single metric. I'm not sure what the best practice is to be honest. My gut feeling tells me not to mix counters and gauges that have a completely different scale. That said, we can have multiple graphs based on a single gauge each grouped by certain labels. I'm not sure it really matters that much. Most people will use the standard dashboard.